### PR TITLE
first cut of documentatin for tasks as they stand now in Nexus OSS

### PR DESCRIPTION
**Pull request by mosabua**
https://issues.sonatype.org/browse/NEXUS-8373

### CODE REVIEW

**Comment by mosabua** [comment]
CI  http://bamboo.s/browse/BOOK-NEXUS13-1

**Comment by mosabua** [comment]
Rendered output at http://bamboo.s/browse/BOOK-NEXUS13-1/artifact/STAGE/site/3.0/reference/admin-system.html#tasks

File: chapter-configuration.asciidoc [hunk 1]
```diff
@@ -184,6 +184,306 @@ Hosts' setting. <<fig-admin-system-http-proxy>> shows the 'HTTP Proxy'
...
+log files.
+
+Type:: The type of action the scheduled task executes. The list of
+available task types is documented in more detail below.
+
+Status:: Tasks can either be 'Waiting' for their next run, currently
+'Running' or 'Disabled'.
+
+Schedule:: The 'Schedule' column shows the 'Task frequency' e.g.,
+'Daily', 'Monthly' , 'Manual' and others.
```
Comment (joedragons) [inline chapter-configuration.asciidoc]: Minor: Extra space after 'Monthly' but before the comma.

File: chapter-configuration.asciidoc [hunk 2]
```diff
@@ -184,6 +184,306 @@ Hosts' setting. <<fig-admin-system-http-proxy>> shows the 'HTTP Proxy'
...
+Notification Email:: Configure a notification email for task execution
+failures. If a scheduled task fails a notification email containing
+the task identifier and name as well as the stack trace of the failure
+will be sent to the configured email recipient. 
+
+Task frequency:: Selecting the task frequencey allows you to configure
+the schedule for the task executions. Available choices are 'Manual',
+'Once', 'Hourly', 'Daily', 'Weekly', 'Monthly' and 'Advanced (provide
+a CRON expression'. All choices trigger display of a custom user
+interface for scheduling the specific recurrence. Weekly scheduling
```
Comment (joedragons) [inline chapter-configuration.asciidoc]: Closing ) of "Advanced (provide a CRON expression)" is missing.

Minor: Manual does not trigger a custom display, however, I can't think of a clean way to write it without specifically saying that or being incrediably verbose, so may leave this alone.

**Comment by joedragons** [comment]
+1

File: chapter-configuration.asciidoc (hunk 1 above)
Comment (mosabua) [inline chapter-configuration.asciidoc]: Fixed

File: chapter-configuration.asciidoc (hunk 2 above)
Comment (mosabua) [inline chapter-configuration.asciidoc]: Fixed both as well.

**Pull request merged**